site/projects/isentropic_analysis.md

```diff
@@ -1,14 +1,16 @@
 Isentropic Analysis Case Study
 ==============================
 
-![under construction](https://images2.minutemediacdn.com/image/upload/c_fit,f_auto,fl_lossy,q_auto,w_728/v1555999902/shape/mentalfloss/under_construction1_0.gif?itok=Pn9g_wu6)
+[comment]: <> (![under construction]&#40;https://images2.minutemediacdn.com/image/upload/c_fit,f_auto,fl_lossy,q_auto,w_728/v1555999902/shape/mentalfloss/under_construction1_0.gif?itok=Pn9g_wu6&#41;)
 
 ## Description
 
-_description here_
+Create plots for using isentropic coordinates within synoptic analysis and forecasting.
 
 ## Related Training Notebooks
 
+[Isentropic Introduction](notebooks/visualization/isentropic_introduction.ipynb)
```

Comment:
Looks like the build docs check is failing on this line. We've been referencing links to notebooks using nbviewer, but it's being a bit flaky at the moment...and is making me rethink that choice :-/

Reply:
Hmm...I updated the link and there's still a problem. Seems to still fail at the same line, don't really know what the problem is.